Lib/fontTools/feaLib/variableScalar.py

```diff
@@ -95,3 +95,66 @@ def add_to_variation_store(self, store_builder):
         store_builder.setSupports(supports)
         index = store_builder.storeDeltas(deltas)
         return int(self.default), index
+
+    def __sub__(self, other):
```

Comment:
the canonical way for magic operator overrides is to first check if other is not an instance of the expected type, `return NotImplemented` so that the other operand has the chance to implement the operation:

```python
if not isinstance(other, VariableScalar):
    return NotImplemented
```